CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### BREAKING CHANGES
 
+* Removes AWS and Azure resource detectors.
```

Comment:
in favor of upstream?

Reply:
Instead of bundling the detectors and enabling then by default, the detector packages will need to be installed and then enabled via code or env vars.

The cloud platform detectors all fall back on DNS to determine their environment, and DNS timeouts on some platforms (GCP) can be 5+ minutes.

There have also been issues with detectors throwing exceptions when activated outside the expected environment. Some AWS packages were erroring out assuming certain environment variables would always exist.